### ISSUE TITLE
Size of the dictionary

### ISSUE THREAD

**Issue by sordonia** [synapse-developpement/Discovery#1]
Hi,

Thanks for the dataset and the interesting paper. I was wondering if you had released the code to reproduce the experiments ? Also, are you using the full Glove vocabulary for the experiments ?

Thanks a lot

**Comment by sileod** [comment]
Hi, thank you for your interest ! 
I use the off the shelf InferSent library. In this library, the Glove embeddings are used without finetuning. The vocabulary is built at the base task training step (e.g. NLI ou Discovery) and rebuilt at evaluation steps (for all words that appear once and are in the glove vocabulary, the glove vector is used, else the word is skipped)

I might try to release a pickle infersent model trained on Discovery; But the replication is rather straightforward, using this library https://github.com/facebookresearch/InferSent and replacing AllNLI ou SNLI by Discovery (the format is the same, you just have to take care of the change in the labels)
